### ISSUE TITLE
Error: Could not find the tree hash of run

### ISSUE THREAD

**Issue by PaintingWithCode** [fkirc/skip-duplicate-actions#127]
Hi there, we've been using this action for the last couple of months without any issues, but sometime yesterday the action has started failing ~75% of the time with the following output:
```
Error: Could not find the tree hash of run [object Object]
```

For context, we use this action to check for PR changes to a given service directory within our monorepo, and if no changes are found, it skips the corresponding required CI job. Altogether the action is concurrently run on four separate workflows per PR.

We are seeing that error pop up and fail the run, seemingly at random, across all four workflows on multiple PRs. Our current work around for this has been to manually re-trigger the workflow till the tree hash is found and the workflow executes without a crash, but it isn't a viable solution in the long run. 

Any idea what could be causing it? 

Here's a sample log of one of the failure runs:
```
Current runner version: '2.279.0'
Operating System
  Ubuntu
  20.04.2
  LTS
Virtual Environment
  Environment: ubuntu-20.04
  Version: 20210803.0
  Included Software: https://github.com/actions/virtual-environments/blob/ubuntu20/20210803.0/images/linux/Ubuntu2004-README.md
  Image Release: https://github.com/actions/virtual-environments/releases/tag/ubuntu20%2F20210803.0
Prepare workflow directory
Prepare all required actions
Getting action download info
Download action repository 'fkirc/skip-duplicate-actions@master'
------------------------------------------------------------------------------------
Run fkirc/skip-duplicate-actions@master
  with:
    paths: ["server/**"]
    paths_ignore: ["**/README.md"]
    do_not_skip: ["push"]
    github_token: ***
    cancel_others: false
    skip_after_successful_duplicate: true
    concurrent_skipping: never
    
Error: Could not find the tree hash of run [object Object]
```

**Comment by fkirc** [comment]
I do not know why this happens, but perhaps it is a viable workaround to add `continue-on-error: true` to your pre-jobs.

**Comment by nsavelyeva** [comment]
Hello,
I also caught this issue today in different PRs and it seems that re-running jobs does not help.
Is the root cause of this still unknown?

**Comment by paescuj** [comment]
This is now fixed by #178, right?

**Comment by paescuj** [comment]
Closing this issue as it is very likely that it has been fixed by https://github.com/fkirc/skip-duplicate-actions/pull/178. Please open a new issue if there is still a problem.
